### DISCUSSION TITLE
Multiple artists

### DISCUSSION THREAD

**Discussion by baloe** [MusicPlayerDaemon/MPD#1385] [Q&A] [answered]
Are multiple artists supported? I currently have multiple artists separated by semicolons like so:

    Album-artist: "One Artist"
    Artist: "One Artist; Other Artist; Third Artist"

When listing by artist I would then expect to see the three artists appear in the list separately.

Other music playing software that I use support this format. 
My files are stored as FLACs.

**Comment by MaxKellermann** [comment, marked as answer]
Yes, you can have multiple "Artist" tags, and MPD supports that well.
Your example however is just one "Artist" tag, with just one value. You stuffed multiple strings separated by semicolons in there, but MPD doesn't recognize that as multiple values; it is technically just one value.

**Reply by baloe** [reply]
ok. I will write a script then to clean up my metadata, separating the single string containing multiple artists into separate artist tags each containing one artist. Thanks for the fast response!

**Reply by MPconta** [reply]
You can use for example Mp3tag for this job. It's quite easy.

**Reply by not4smurf** [reply]
I know this is an old thread, but it seems more appropriate to continue here rather than start a new one.

I'm in the process of tidying my library with MusicBrainz Picard.  Picard writes just one ARTIST (ID3v2 TPE1 or Vorbis ARTIST) tag and, if required, multiple ARTISTS (ID3 TXXX:ARTISTS or Vorbis ARTISTS) tags.  MPD only allows the "Artist" (singular) tag. 

If an issue was raised to have MPD also support the "artists" tag, might it be implemented at some point?

**Reply by 2084x** [reply]
interestingly multiple artist tags seems to be supported by mpd but not mpc, which only shows the first tag.

**Comment by Zorgodon** [comment]
Separating multiple artists with semicolons *is* however the standard way of doing it. I've used everything from foobar2000 to Swinsian to gtkpod and MPD is the first piece of software I've seen get confused by this.
